src/Nethermind/Nethermind.Blockchain/BlockTree.cs

```diff
@@ -43,8 +43,8 @@ public partial class BlockTree : IBlockTree
     {
         private const long LowestInsertedBodyNumberDbEntryAddress = 0; 
         private const int CacheSize = 64;
-        private readonly ICache<Keccak, Block> _blockCache = new LruCacheWithRecycling<Keccak, Block>(CacheSize, CacheSize, "blocks");
-        private readonly ICache<Keccak, BlockHeader> _headerCache = new LruCacheWithRecycling<Keccak, BlockHeader>(CacheSize, CacheSize, "headers");
+        private readonly ICache<Keccak, Block> _blockCache = new LruCache<Keccak, Block>(CacheSize, CacheSize, "blocks");
```

Comment:
why not recycling? I Lru cache now recycling?

Reply:
yes, LruCache was not used anywhere so I renamed recycling to normal and removed the other one